### PR TITLE
#3983 [Manager] Reset Workspace

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3983 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Workspace/Edit/MoreActions.js [hunk 1]
```diff
@@ -47,9 +126,31 @@ function MoreActions() {
         open={Boolean(anchorEl)}
         onClose={handleClose}
       >
-        <MenuItem onClick={handleClose}>ADD PLUGINS</MenuItem>
-        <MenuItem onClick={handleClose}>ADD NODES</MenuItem>
+        <MenuItem onClick={() => setIsOpne(true)}>RESET WORKSPACE</MenuItem>
```
Comment (eechih) [inline ohara-manager/client/src/components/Workspace/Edit/MoreActions.js]: UI 出現 reset 和 restart 兩個 term，避免使用者混淆，請盡量統一名詞。

Comment (wu87988622) [reply ohara-manager/client/src/components/Workspace/Edit/MoreActions.js]: ok~ thanks.

File: ohara-manager/client/src/components/Workspace/Edit/MoreActions.js [hunk 2]
```diff
@@ -47,9 +129,32 @@ function MoreActions() {
...
+        false && <MenuItem onClick={handleClose}>ADD PLUGINS</MenuItem>}
+        {/* Feature is disabled because it's not implemented in 0.9 */
+        false && <MenuItem onClick={handleClose}>ADD NODES</MenuItem>}
       </Menu>
+      <Dialog
+        open={isOpen}
+        title={'Reset Workspace?'}
+        children={
+          <Typography>
+            Are you sure you want to reset the workspace, This action cannot be
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/Edit/MoreActions.js]: I think we should probably warn and tell users what we're about to do either in this dialog or the action button? This just ensure that the users "understand" what this action is about to do :)

File: ohara-manager/client/src/components/Workspace/Edit/MoreActions.js [hunk 3]
```diff
@@ -47,9 +129,32 @@ function MoreActions() {
...
+            undone!
+          </Typography>
+        }
+        confirmText={'RESET'}
+        handleClose={handleDialogClose}
+        handleConfirm={() => reset()}
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/Edit/MoreActions.js]: I don't see why we need to create another anonymous function here

```suggestion
        handleConfirm={reset}
```

**Comment by wu87988622** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/Workspace/Edit/MoreActions.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Workspace/Edit/MoreActions.js]: ok thans

File: ohara-manager/client/src/components/Workspace/Edit/MoreActions.js (hunk 3 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Workspace/Edit/MoreActions.js]: oh oh nice !!!

**Pull request merged**